Gemfile

```diff
@@ -33,7 +33,7 @@ end
 puppetversion = ENV.key?('PUPPET_VERSION') ? "#{ENV['PUPPET_VERSION']}" : ['>= 3.3']
 gem 'puppet', puppetversion
 
-facterversion = ENV.key?('FACTER_GEM_VERSION') ? "#{ENV['FACTER_GEM__VERSION']}" : ['>= 1.7.0']
+# facterversion = ENV.key?('FACTER_GEM_VERSION') ? "#{ENV['FACTER_GEM__VERSION']}" : ['>= 1.7.0']
```

Comment:
Why did this get commented out.  I suspect we might not need this if it's just a carry over from our standalone module.

Reply:
We're just using 
```code
gem 'facter', '>= 1.7.0'
```

Removing for now, we can add it back later